fluentui_core/src/main/java/com/microsoft/fluentui/compose/Scaffold.kt

```diff
@@ -0,0 +1,305 @@
+package com.microsoft.fluentui.compose
+
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
```

Comment:
Is this needed?

Reply:
Yes as opensource code used directly & modified as per our requirements.